modules/react/avatar/stories/avatar/examples/ObjectFit.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {Avatar} from '@workday/canvas-kit-react/avatar';
+import {px2rem} from '@workday/canvas-kit-styling';
+
+export const ObjectFit = () => (
+  <div className="story">
+    <h3>Original Rectangle Image</h3>
+    <img
+      alt=""
+      src="https://loremflickr.com/cache/resized/65535_53325191273_373deb18ee_n_320_200_nofilter.jpg?lock=1"
+    />
+    <h3>Object fit on a Rectangle Image</h3>
+    <Avatar
+      as="div"
+      url="https://loremflickr.com/cache/resized/65535_53325191273_373deb18ee_n_320_200_nofilter.jpg?lock=1"
```

Comment:
The image for some reason isn't showing up, can you try this image?
https://loremflickr.com/cache/resized/65535_52850844031_00cab311b5_n_250_180_nofilter.jpg